<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_serveurClass</class>
 <widget class="QMainWindow" name="Client_serveurClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client_serveur</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Client</string>
    </property>
    <widget class="QLineEdit" name="leClientAdresse">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>44</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClientConnect">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClientSendMess">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>444</y>
       <width>141</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer Message</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leClientMessToSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="teClientReception">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reception :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leClientPort">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>44</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>9090</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Port :</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Serveur</string>
    </property>
    <widget class="QLineEdit" name="leServerPort">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>44</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>9090</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Ã©coute :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbServerConnect">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbServerSendMess">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>440</y>
       <width>141</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer Message</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServerMessToSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>446</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="teServerReception">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reception :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client_serveur.qrc"/>
 </resources>
 <connections/>
</ui>
